core/designsystem/src/main/kotlin/com/google/samples/apps/nowinandroid/core/designsystem/component/Navigation.kt

```diff
@@ -165,6 +182,116 @@ fun NiaNavigationRail(
+ * @param navigationSuiteItems A slot to display multiple items via [NiaNavigationSuiteScope].
+ * @param windowAdaptiveInfo The window adaptive info.
+ * @param content The app content inside the scaffold.
+ */
+@OptIn(
+    ExperimentalMaterial3AdaptiveNavigationSuiteApi::class,
+    ExperimentalMaterial3AdaptiveApi::class,
+)
+@Composable
+fun NiaNavigationSuiteScaffold(
+    navigationSuiteItems: NiaNavigationSuiteScope.() -> Unit,
+    modifier: Modifier = Modifier,
+    windowAdaptiveInfo: WindowAdaptiveInfo = currentWindowAdaptiveInfo(),
+    content: @Composable () -> Unit,
+) {
+    val layoutType = NavigationSuiteScaffoldDefaults
+        .calculateFromAdaptiveInfo(windowAdaptiveInfo)
+    val navigationSuiteItemColors = NavigationSuiteItemColors(
+        navigationBarItemColors = NavigationBarItemDefaults.colors(
+            selectedIconColor = NiaNavigationDefaults.navigationSelectedItemColor(),
+            unselectedIconColor = NiaNavigationDefaults.navigationContentColor(),
+            selectedTextColor = NiaNavigationDefaults.navigationSelectedItemColor(),
+            unselectedTextColor = NiaNavigationDefaults.navigationContentColor(),
+            indicatorColor = NiaNavigationDefaults.navigationIndicatorColor(),
+        ),
+        navigationRailItemColors = NavigationRailItemDefaults.colors(
+            selectedIconColor = NiaNavigationDefaults.navigationSelectedItemColor(),
+            unselectedIconColor = NiaNavigationDefaults.navigationContentColor(),
+            selectedTextColor = NiaNavigationDefaults.navigationSelectedItemColor(),
+            unselectedTextColor = NiaNavigationDefaults.navigationContentColor(),
+            indicatorColor = NiaNavigationDefaults.navigationIndicatorColor(),
+        ),
+        navigationDrawerItemColors = NavigationDrawerItemDefaults.colors(
+            selectedIconColor = NiaNavigationDefaults.navigationSelectedItemColor(),
+            unselectedIconColor = NiaNavigationDefaults.navigationContentColor(),
+            selectedTextColor = NiaNavigationDefaults.navigationSelectedItemColor(),
+            unselectedTextColor = NiaNavigationDefaults.navigationContentColor(),
+        ),
+    )
+
+    NavigationSuiteScaffold(
+        navigationSuiteItems = {
+            NiaNavigationSuiteScope(
+                navigationSuiteScope = this,
+                navigationSuiteItemColors = navigationSuiteItemColors,
+            ).run(navigationSuiteItems)
+        },
+        layoutType = layoutType,
+        containerColor = Color.Transparent,
+        navigationSuiteColors = NavigationSuiteDefaults.colors(
+            navigationBarContentColor = NiaNavigationDefaults.navigationContentColor(),
+            navigationRailContainerColor = Color.Transparent,
+        ),
+        modifier = modifier,
+    ) {
+        Box(
+            // TODO: It'd be nice if navigation suite scaffold did this automatically:
+            //       https://issuetracker.google.com/334178393
```

Comment:
This is fixed in the next release, so maybe create an issue in the nowinandroid repo to update to that version and delete this custom code.

Reply:
Removed this workaround with the latest version of the library.